### PR TITLE
Added dynamic variables to templates

### LINKED ISSUE #1073: Make sure all templates are using the new dynamic variables
For 
PROJECT_NAME
DOMAIN_NAME
FQDN
There are about 38 templates to replace the word “bugheist” in with these new variables

### PR DESCRIPTION
**Pull request by Kej-r03**
Fixes #1073 
Added custom template tag  'env' to use the settings.py variables inside templates

### CODE REVIEW

**Review by fredfalcon — DISMISSED** [review]
Looks good.  A few tweaks required

File: website/templates/account/login.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
           <a href="{% url 'index' %}" class="hover:no-underline text-white hover:text-white">BUGHEIST</a>
         </div>
         <p class="mt-6 font-normal text-center text-white text-xl md:mt-0">
-            OWASP BLT is a bug logging tool to report issues and get points, companies are held accountable.
+            OWASP {% env 'PROJECT_NAME' %} is a bug logging tool to report issues and get points, companies are held accountable.
```
[the remark this answers is not in the document]
Comment (Kej-r03) [reply website/templates/account/login.html]: Done

File: website/templates/account/signup.html [hunk 2]
```diff
@@ -14,7 +14,7 @@
           <a href="{% url 'index' %}" class="hover:no-underline text-white hover:text-white">BUGHEIST</a>
         </div>
         <p class="mt-6 font-normal text-center text-white text-xl md:mt-0">
-            OWASP BLT is a bug logging tool to report issues and get points, companies are held accountable.
+            OWASP {% env 'PROJECT_NAME' %} is a bug logging tool to report issues and get points, companies are held accountable.
```
[the remark this answers is not in the document]
Comment (Kej-r03) [reply website/templates/account/signup.html]: Done

File: website/templates/domain.html [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {% extends "base.html" %}
-{% load issue_type %}
+{% load custom_tags %}
```
[the remark this answers is not in the document]
Comment (Kej-r03) [reply website/templates/domain.html]: No, I have removed it in the following commit

File: website/templates/includes/footer.html [hunk 4]
```diff
@@ -9,7 +10,7 @@
         <div class="w-2/3 block sm:flex text-2xl mt-6 lg:mt-0">
             <ul class="text-gray-700 list-none p-0 font-thin flex flex-col text-left w-full">
                 <li class="inline-block py-2 px-3 text-white uppercase font-medium tracking-wide">Company</li>
-                <li><a href="https://github.com/OWASP/BLT/releases" class="inline-block py-2 px-3 text-gray-500 hover:text-white no-underline">Features</a></li>
+                <li><a href="https://github.com/OWASP/{% env 'PROJECT_NAME' %}/releases" class="inline-block py-2 px-3 text-gray-500 hover:text-white no-underline">Features</a></li>
```
[the remark this answers is not in the document]
Comment (Kej-r03) [reply website/templates/includes/footer.html]: Rectified in the following commit

File: website/templates/includes/footer.html [hunk 5]
```diff
@@ -21,9 +22,9 @@
             </ul>
             <ul class="text-gray-700 list-none p-0 font-thin flex flex-col text-left w-full">
                 <li class="inline-block py-2 px-3 text-white uppercase font-medium tracking-wide">Get Started</li>
-                <li><a href="https://www.bugheist.com/swagger/" class="inline-block py-2 px-3 text-gray-500 hover:text-white no-underline">Developer API</a></li>
-                <li><a href="https://github.com/OWASP/BLT/blob/main/Setup.md" class="inline-block py-2 px-3 text-gray-500 hover:text-white no-underline">Documentation</a></li>
-                <li><a href="https://github.com/OWASP/BLT#readme" class="inline-block py-2 px-3 text-gray-500 hover:text-white no-underline">Guides</a></li>
+                <li><a href="https://{% env 'FQDN' %}/swagger/" class="inline-block py-2 px-3 text-gray-500 hover:text-white no-underline">Developer API</a></li>
+                <li><a href="https://github.com/OWASP/{% env 'PROJECT_NAME' %}/blob/main/Setup.md" class="inline-block py-2 px-3 text-gray-500 hover:text-white no-underline">Documentation</a></li>
```
[the remark this answers is not in the document]
Comment (Kej-r03) [reply website/templates/includes/footer.html]: Rectified in the following commit

**Comment by Kej-r03** [comment]
Hi @fredfalcon, I have addressed all the reviews and changed accordingly

**Review by fredfalcon — DISMISSED** [review]

**Review by fredfalcon — APPROVED** [review]

**Pull request merged**